velox/functions/prestosql/registration/GeneralFunctionsRegistration.cpp

```diff
@@ -48,8 +59,19 @@ void registerGeneralFunctions(const std::string& prefix) {
   VELOX_REGISTER_VECTOR_FUNCTION(udf_reduce, prefix + "reduce");
   VELOX_REGISTER_VECTOR_FUNCTION(udf_array_filter, prefix + "filter");
 
-  VELOX_REGISTER_VECTOR_FUNCTION(udf_least, prefix + "least");
-  VELOX_REGISTER_VECTOR_FUNCTION(udf_greatest, prefix + "greatest");
+  registerGreatestLeastFunction<bool>(prefix);
```

Comment:
nit: perhaps, extract this code into registerGreatestLeastFunctions(prefix) helper function; otherwise, as more functions are added in the same manner, this method will become very large.